librz/include/rz_userconf.h.in

```diff
@@ -48,6 +48,8 @@
 #define HAVE_COPYFILE               @HAVE_COPYFILE@
 #define HAVE_COPY_FILE_RANGE        @HAVE_COPY_FILE_RANGE@
 #define HAVE_BACKTRACE              @HAVE_BACKTRACE@
+#define HAVE_GETPID                 @HAVE_GETPID@
+#define HAVE__GETPID                @HAVE__GETPID@
```

Comment:
why sdb needs getpid ?

Reply:
it does for now. not exactly sure why, but we can remove it later if not strictly necessary.